src/sentry/receivers/onboarding.py

```diff
@@ -164,6 +164,7 @@ def record_first_event(project, event, **kwargs):
         organization_id=project.organization_id,
         project_id=project.id,
         platform=event.platform,
+        project_platform=project.platform,
```

Comment:
@ceorourke still need to update how we generate the `first_event.sent` analytics event. Pretty sure this is going to blow up without that change.

Reply:
It does:

> FAILED tests/sentry/receivers/test_onboarding.py::OrganizationOnboardingTaskTest::test_onboarding_complete - ValueError: project_platform is required (cannot be None)

Reply:
Oh is `first_event.sent` supposed to have the `project_platform` attribute too? I think I misread the ticket which is why I put it only for the other 2.

Reply:
@ceorourke Yep, that's the primary one